intel_extension_for_transformers/llm/runtime/graph/core/layers/Ops.h

```diff
@@ -85,6 +85,7 @@ enum ne_op {
   NE_OP_ALL_REDUCE,
   NE_OP_TP_CONCAT,
   NE_OP_DUMP_TENSOR,
+  NE_OP_DBG,
```

Comment:
DBG for ?

Reply:
Debug

Reply:
OK, it's reasonable to add back the 2 letters for clarity.